tests/Test+Shape+Estimation.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
Interesting to have tests in a notebook - we must get travis to run them for us.

Reply:
This test is a bit outdated and more of a practice notebook for me for visualization purposes -- should remove it?

Reply:
Let's leave it in for now - it's a good idea to have notebooks as system tests.